app/cdap/components/AddTransformation/CircularProgressBar/utils.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IHeaderNamesList, IDataQuality, IRecords } from '../types';
+
+export const prepareDataQualtiy = (statistics: IDataQuality, columnList: IHeaderNamesList[]) => {
```

Comment:
Is this IDataQuality reused somewhere for quality?
If yes, then can we rename it to generalized one, as it suits for both Data Quality and Statistics, so IStatistics makes more sense as Statistics is an entity and data quality is generated by us. So, we are saying DataQuality should be structured according to IStatistics Object. :) Hope this makes sense. Please update accordingly.
If no, then can we rename it to IStatistice, I think IStatistics already exists, plz have a look otherwise create one